README.md

```diff
@@ -55,6 +55,7 @@ Command           | Package Id | Author | Description
 `dotnet-script` | [dotnet-script](https://www.nuget.org/packages/dotnet-script/) | [@filipw](https://github.com/filipw) [@seesharper](https://github.com/seesharper) | Run C# scripts from the .NET CLI.  [GitHub](https://github.com/filipw/dotnet-script)
 `dotnet-search` | [dotnet-search](https://www.nuget.org/packages/dotnet-search/) | [@billpratt](https://github.com/billpratt) | Search for Nuget packages using the .NET Core CLI.  [GitHub](https://github.com/billpratt/dotnet-search)
 `dotnet-serve` | [dotnet-serve](https://www.nuget.org/packages/dotnet-serve/) | [@natemcmaster](https://github.com/natemcmaster) | A simple command line HTTP server, no code required.  [GitHub](https://github.com/natemcmaster/dotnet-serve)
+`dotnet-snow` | [dotnet-snow](https://www.nuget.org/packages/dotnet-snow/) | [@natemcmaster](https://github.com/ptupitsyn) | Avalonia-based cross-platform graphical snow demo.  [GitHub](https://github.com/ptupitsyn/let-it-snow)
```

Comment:
I think you forgot to change the actual text of your profile link! @ptupitsyn 😅

Reply:
Oh, you are right :)